packages/matrix-identity-server/src/cron/index.ts

```
@@ -88,108 +66,72 @@ class CronTasks<T extends string = never> {
-   */
-  private readonly _addCheckUserQuotaJob = async (
-    conf: Config,
-    db: IdentityServerDb<T>,
-    logger: TwakeLogger
-  ): Promise<void> => {
-    const cronString = conf.check_quota_cron ?? '0 0 0 * * *'
-
-    if (!cron.validate(cronString)) {
-      // istanbul ignore next
-      throw new Error(`Invalid cron line: ${cronString}`)
+      throw Error("Failed to add update hashes job");
```

Biome (2.4.9)

[error] Use new Error() instead of Error() when throwing an error.

(lint/style/useThrowNewError)